lib/Runtime/Library/JavascriptBigIntObject.h

```diff
@@ -0,0 +1,35 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+#pragma once
+
+namespace Js
+{
+    class JavascriptBigIntObject : public DynamicObject
+    {
+    private:
+        Field(JavascriptBigInt*) value;
+
+        DEFINE_VTABLE_CTOR(JavascriptBigIntObject, DynamicObject);
+        DEFINE_MARSHAL_OBJECT_TO_SCRIPT_CONTEXT(JavascriptBigIntObject);
+    public:
+        JavascriptBigIntObject(JavascriptBigInt* value, DynamicType * type);
+        static bool Is(Var aValue);
+        static JavascriptBigIntObject* FromVar(Js::Var aValue);
+        static JavascriptBigIntObject* UnsafeFromVar(Js::Var aValue);
+
+        JavascriptBigInt* GetValue() const;
+        void Initialize(JavascriptBigInt* value);
```

Comment:
I think `Initialize` can be safely removed. I tried removing it from `JavascriptBooleanObject`, and it built and ran unit tests fine.